Add debian to setup and podman (#35)
ansible/receptor-collection: README.md

---

---

    local_receptor: false

> Put it to true if you want to use a local receptor bin (or download it from github) instead of package release for your distro.

Reviewer: ```suggestion
Set to true to upload a local receptor binary instead of installing via a package manager. If no local binary is present, it will be downloaded from receptor Releases on GitHub.
```

For Debian we set it a true as default.

---

---

    local_receptor_path: '/tmp/receptor-bin'

> Path where is stored receptor bin file.

Reviewer: ```suggestion
Path of local receptor binary.
```

---

---

    receptor_install_dir: '/usr/bin'

Path where receptor bin will be moved.

---

---

    receptor_path_log: '/var/log/receptor'

> Path where receptor log will be sotred. Used only when local_receptor is true.

Reviewer: ```suggestion
Receptor log directory. Used only when local_receptor is true.
```

---

---

    receptor_github_owner: 'ansible'

Owner of github repo where we search and download package if local_receptor_path not exists.

---

---

    receptor_github_repo: 'receptor'

Repository github where we search and download package if local_receptor_path not exists.

---

---

    receptor_github_release: # not set, if set we use it for download package

Repository version where we search and download package if local_receptor_path not exists.

---

---

    receptor_service_name: 'receptor'

> Name of service inside the OS. Used only in case local_receptor is true and if you want multiple service running on OS.

Reviewer: ```suggestion
Name of systemd service that runs receptor. Used only when local_receptor is true.
```
